Documentation/Supported Features.md

```diff
@@ -83,42 +116,121 @@ This is the set of motions which are currently supported listed alphabetically.
 - O
 - p
 - P
+- q : macro recording
+- r
+- R : Replace mode
 - s
 - S
+- u
+- U
+- v
+- V
 - x
 - X
+- Y
+- yy
+- za
+- zA
+- zo
+- zO
+- zc
+- zC
+- zd
+- zD
+- zE
+- zF
+- zM
+- zR
+- ZQ
+- ZZ
+- &lt;C-a&gt;
```

Comment:
It might involve a lot of work, but we could avoid all of the entities if we used markdown backquote notation systematically.